### DISCUSSION TITLE
Controller Only Mode

### DISCUSSION THREAD

**Discussion by jmagar** [LizardByte/community#285] [Sunshine]
Often times I'll use steam link to use my deck as a controller while playing something from my gaming VM hooked up to my TV. If I want to free up the TV for my partner but keep playing on the deck, I have to close steam link and then start streaming on the deck.

A shortcut to toggle the video stream on/off could make this transition seemless and remove the need to have steam link installed.

**Comment by ReenigneArcher** [comment]
Thank you for your suggestion. At this time, we do not plan to support this. Sunshine is not a USB/IP protocol. You might want to check out one of the many applications designed for that.

**Reply by ReenigneArcher** [reply]
Actually there is a draft PR: https://github.com/LizardByte/Sunshine/pull/1445

No guarantees this will be added though.

**Reply by ReenigneArcher** [reply]
This PR, unfortunately went stale.

**Comment by willow-sprite** [comment]
I second this, sometimes I get sick of hearing my mouseclicks and use my phone as a touchpad.

kdeconnect is the only other equivalent software for linux, but it isn't as seamless as using moonlight for mouse control, but the overhead of running moonlight just for mouse input seems silly.

I wish sunshine had modular components, like getting input from one device, sending audio to another, and sending display only to a final device.

I hope someone picks this up.
